README.md

```diff
@@ -10,7 +10,9 @@ A versatile IRC bouncer.
   it from your distro's repositories
 - Create a Python3 virtualenv: ``mkvirtualenv ircb --no-site-packager -p /usr/bin/python3``
 - Switch to your virtualenv: ``workon ircb``
-- Install dependencies: ``pip install -r requirements.txt``
+- Install dependencies: ``pip install -r requirements.txt``. Incase the
+  installation is not done inside an environment, then install the dependencies
```

Comment:
@sayanchowdhury could you do the above change and remove ``virtualenv`` related instructions?